packages/sql-mapper/test/entity.test.js

```diff
@@ -267,6 +267,29 @@ test('[SQLite] throws if PK is not INTEGER', { skip: !isSQLite }, async ({ fail,
   }
 })
 
+test('[SQLite] throws if PK is not NUMBER', { skip: !isSQLite }, async ({ fail, equal, teardown, rejects }) => {
+  async function onDatabaseLoad (db, sql) {
+    await clear(db, sql)
+    await db.query(sql`CREATE TABLE pages (
+      id NUMERIC PRIMARY KEY,
```

Comment:
with [reference](https://www.sqlite.org/datatype3.html) the type names, seems like `NUMERIC` fits. please advice if otherwise